code/src/lidar_distance_sensor_c.cpp

```diff
@@ -0,0 +1,24 @@
+#include "lidar_distance_sensor_c.hpp"
+
+namespace r2d2::distance
+{
+    void lidar_distance_sensor_c::fill_distance_frame(frame_distance_s& frame){
+
+        frame.sensor_type = 0; // 0 = lidar, 1 = sonar
+        frame.sensor_pov = 0;
+        frame.count = 45;
+
+        // 16 packets for one 360 degree measurement
+        for (int i = 0; i < 16; i++) {
```

Comment:
I get where the 16 is for but now this is a magic number. can this be in a centralized place?

Reply:
that might make it more unclear, imo